packages/breadcrumbs/index.js

```diff
@@ -1,25 +1,48 @@
 import { html, LitElement, css } from 'lit';
 import { interleave } from '@warp-ds/core/breadcrumbs';
+import { breadcrumbs as ccBreadcrumbs } from '@warp-ds/component-classes'
+import { kebabCaseAttributes } from '../utils';
 
-const separator = html`<span class="select-none" aria-hidden="true">/</span>`;
+const separator = html`<span class=${ccBreadcrumbs.separator} aria-hidden='true'>/</span>`;
```

Comment:
Shouldn't this be with double quotes?
```suggestion
const separator = html`<span class="${ccBreadcrumbs.separator}" aria-hidden="true">/</span>`;
```

Reply:
As far as I checked, it's not necessary to have quotes when passing a JS value to the attributes in Web Components. 

<img width="478" alt="Screenshot 2023-04-25 at 08 27 45" src="https://user-images.githubusercontent.com/41303231/234192206-7384f52f-5e13-4f18-b5a3-fa59b6b51344.png">

As for single vs. double quotes, there are no eslint rules set in this repo yet, so the inconsistency is in more files. If we want to fix this, maybe we could do it as a separate task? 😊